src/main/java/core/basesyntax/Plane.java

```diff
@@ -7,5 +7,74 @@
  */
 
 public class Plane {
+    private int planeID;
+    private String planeName;
+    private boolean isInAir;
+    private String pilotName;
+    private int gasCapacity;
 
+    public Plane(PlaneBuilder planeBuilder) {
```

Comment:
https://mate-academy.github.io/jv-program-fulltime/01_core/checklist/hw_05.html
check your solution according to checklist